docs/src/pages/index.mdx

```diff
@@ -0,0 +1,178 @@
+---
+layout: ../layouts/wiki.astro
+---
+
+import InstallCommands from "@/components/install-commands.astro";
+
+# Remeda
+
+Fully written in TypeScript for powerful, type-safe, and functional utilities.
+
+<InstallCommands />
+
+## 🧰 Expansive
+
+Remeda offers a wide range of common everyday utility functions so that you
+don't have to deal with the minutiae of writing them yourself. We ensure that
+our algorithms are efficient, optimized, and cover all edge cases properly. We
```

Comment:
**question:** i don't think we can say that our algorithms are "optimized" given that we don't run any benchmarks?

i can accept "efficient" though, given that stuff like `takeFirstBy` is n log n rather than n^2 or something

Reply:
I agree, I'll change it